avocado/utils/softwareraid.py

```diff
@@ -165,3 +165,19 @@ def stop(self):
         """
         cmd = f"mdadm --manage {self.name} --stop"
         return self._run_command(cmd, log_details=False)
+
+    def is_raid_exists(self):
+        """
+        checks if softwareraid exists or not
+
+        :param raid_name: softwareraid name to be checked
```

Comment:
raid_name is not an argument of this method.

Reply:
Done